readthedocs/rtd_tests/tests/test_backend.py

```diff
@@ -1,21 +1,33 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import (
-    absolute_import, division, print_function, unicode_literals)
+    absolute_import,
```

Comment:
If we are deprecating Python2 we should remove these imports and also configure .isort to not add them anymore.

Reply:
I opened https://github.com/rtfd/common/pull/19